index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="navbar">
+        <div>
+            <h3 class="my-logo">My logo</h3>
+        </div>
+
+       <div class="nav-right">
+           <img src="images/Enabled.png" alt="normal button">
+       </div>
+    </nav>
+
+    <section>
+        <h2 class="name">
+            I'm Piesie
+            Glad to see 
+             you!
+        </h2>
+        <p class="about">
+            I'm a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don't 
+            hestiate to contact me.
+        </p>
+
+        <div class="social-media">  
+            <p>LET'S CONNECT</p>
```

Comment:
- [ ] Kindly make sure to use an `<a>` tag or a `<button>` tag for this section, since it's a link it needs to be clickable.

Reply:
@piesieomane that should <h1-h6> tag the `a` or `button` tag is not necessary..

Reply:
please i have done that